src/models/interfaces/IActions.ts

```diff
@@ -37,6 +38,8 @@ export interface IAction {
 	pipeline_id?: string;
 	start_listening?: boolean;
 
+	browser_mod?: any;
```

Comment:
No anys! Looking at your example, we should make a new interface like this:

```typescript
export interface IBrowserMod {
    service?: string;
    data?: IData;
}
```

Or if we're expecting other fields in `browser_mod` we can make something more general using `Record<>`

Reply:
totally makes sense, updated